src/main/java/mekanism/common/tile/prefab/TileEntityInternalMultiblock.java

```diff
@@ -63,16 +63,21 @@ public void onNeighborChange(Block block, BlockPos neighborPos) {
         super.onNeighborChange(block, neighborPos);
         //TODO - V11: Make this properly support changing blocks inside the structure when they aren't touching any part of the multiblocks
         //Note: We handle when an internal multiblock is removed that isn't touching anything in BlockMekanism#onRemove
-        if (!isRemote() && multiblock != null) {
-            //If the neighbor change happened to a block inside a multiblock, and it isn't a block that is part of the multiblock
-            if (level.isEmptyBlock(neighborPos) || !multiblock.isKnownLocation(neighborPos)) {
-                //And we are not already an internal part of the structure, or we are changing an internal part to air
-                // then we mark the structure as needing to be re-validated
-                //Note: This isn't a super accurate check as if a node gets replaced by command or mod with say dirt
-                // it won't know to invalidate it but oh well. (See java docs on internalLocations for more caveats)
-                multiblock.recheckStructure = true;
-            }
-        }
+        if (level == null || multiblock == null || isRemote()) return;
+        //If the neighbor change happened to a block inside a multiblock, and it isn't a block that is part of the multiblock
+        if (!level.isEmptyBlock(neighborPos) && multiblock.isKnownLocation(neighborPos)) return;
+        //If the neighbor change happened to a block outside a multiblock, and it isn't a block that could be part of the multiblock
```

Comment:
Either return the nester `if` statements, or rewrite the comments to make sense (all three sections)